sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -31,10 +31,10 @@
     # 'me-south-1',
     # 'me-central-1',
     # 'af-south-1',
-    # 'af-south-2',
```

Comment:
nit: why is this deleted?

Reply:
There is no` af-south-2`. This should be a typo for `ap-south-2`?

Reply:
Can we fix L33 as well?

Reply:
I think L33 is correct. AWS does have `af-south-1`, but no `af-south-2`.